docs/pages/api-reference/liveblocks-client.mdx

```diff
@@ -587,6 +654,62 @@ const client = createClient({
 });
 ```
 
+#### Group mentions
+
+To support group mentions in [Comments](/docs/ready-made-features/comments) and
+[Text Editor](/docs/ready-made-features/text-editor), you can return a list of
+mention objects instead of user IDs to suggest a mix of user and group mentions.
+
+```tsx
+import { createClient } from "@liveblocks/client";
+
+const client = createClient({
+  resolveMentionSuggestions: async ({ text, roomId }) => {
+    const dbUsers = await __findUsersFromDB__(roomId);
+    const dbGroups = await __findGroupsFromDB__(roomId);
+
+    // Show groups and users matching the text being typed
+    return [
+      ...dbGroups.map((group) => ({
+        kind: "group",
+        id: group.id,
+      })),
+      ...dbUsers.map((user) => ({
+        kind: "user",
+        id: user.id,
+      })),
+    ];
+  },
+
+  // Other options
+  // ...
+});
+```
+
+The mention objects specify which kind of mention it is, the ID to mention (user
+ID or group ID), etc.
+
+```tsx
+// A user mention suggestion
+{
+  kind: "user",
+  id: "user-1",
+}
+
+// A group mention suggestion
+{
+  kind: "group",
+  id: "group-1",
+}
+
+// A group mention suggestion with fixed group members
+{
+  kind: "group",
+  id: "group-1",
+  members: ["user-1", "user-2"],
+}
+```
+
```

Comment:
The way I understand it, the id `"here"` used in other examples is not a special ID. Instead, what makes it special is the presence of a `userIds` key. This makes sense.

But what happens if you do this: `{ kind: "group", id: "actuallyExistingGroupId", userIds: ["a", "b", "c"] }`? I think the local `userIds` take precedence and will cause the group's members not be looked up in that case, right?

If so, that is definitely the behavior I would expect. Maybe good to explicitly document that?